client/src/components/Commits.js

```diff
@@ -18,6 +19,7 @@ class Commits extends React.Component {
           {...commit}
           builds={this.props.build[commit.sha]}
           tests={this.props.testData[commit.sha]}
+          previousTests={this.props.testData[commit.parents[0].sha]}
```

Comment:
Do we need a whole array of previous tests at all? Maybe we should make an object with {testType: testCount} data in it?

Reply:
No, at current moment whole test objects is not required. I will change this behavior.